code/game/objects/items/weapons/tools.dm

```diff
@@ -13,6 +13,15 @@
  */
 
 /*
+ * Used for fancy tool subtypes that are faster or slower than the standard tool.
+ * The number in the first slot is the multiplier for time to construct, the one in the second slot is the multiplier for time to deconstruct.
+ * If one is zero, the tool cannot be used in that direction. If you want to adminbus an instant tool, use .0001 or something, not 0.
+ * Don't set either to a negative number. It will probably break, though I'm not really sure in what way.
+ */
+/atom
+	var/list/construction_delay_mult = list(1, 1)

```

Comment:
you could always make this an associative list to get around the "magic numbers" deal.

So instead of ```construction_delay_mult[1]``` you can do ```construction_delay_mult["construction"]```
Just something to consider.